ITensorGPU/test/runtests.jl

```diff
@@ -1,6 +1,22 @@
-using ITensorGPU, Test, CUDA
+if VERSION < v"1.8"
+  itensorgpu_path = joinpath(pwd(), "..")
+  cp(
+    joinpath(pwd(), "..", "LocalPreferences.toml"),
+    joinpath("$(LOAD_PATH[2])", "JuliaLocalPreferences.toml"),
+  )
+  #old_path = pop!(LOAD_PATH)
+  #push!(LOAD_PATH, itensorgpu_path)
+  #push!(LOAD_PATH, old_path)
```

Comment:
Probably should remove these lines.

Reply:
Okay, this was just another viable solution to the `LocalPreferences` + `Pkg.test` issue in julia v"1.6" and v"1.7"